ckanext/packagezip/helpers.py

```diff
@@ -8,5 +8,10 @@ def has_packagezip(pkg):
 def packagezip_url(pkg):
     return toolkit.url_for('zipfile', name=pkg.name)
 
+def packagezip_size(pkg):
```

Comment:
This is where I'm using it. https://github.com/datagovuk/ckanext-dgu/commit/a7370c0565a652f4b724af44f8ab8fc07a864676

Would it be preferable to return 0?